templates/index.html

```diff
@@ -199,7 +199,7 @@ <h1 class="top-bottom-padding">حمایت‌کنندگان</h1>
 
         <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1 col-sm-12">
             {% for donator in donatores %}
-                <img src="{{ donator.avatar }}" class="avatar" title="{{ donator.name }}">
+                <img src="{{ donator.avatar }}" class="avatar" width="64" height="64" title="{{ donator.name }}">
```

Comment:
width and height will be set automatically by css but if you set them responsive mode will be deactivated!!!
@itmard